cli/state.rs

```diff
@@ -362,6 +357,17 @@ impl ModuleLoader for State {
 
     async { Ok(()) }.boxed_local()
   }
+
+  fn before_evaluate(&self, module_info: &ModuleInfo) {
```

Comment:
rename to `wait_for_inspector()` and call just before `self.isolate.mod_evaluate()` in `Worker`? Then we don't need to bleed this into `core/`. My take: #5230